src/main/java/frc/robot/RobotContainer.java

```diff
@@ -101,7 +107,36 @@ private void configureButtonBindings() {
             swerveSubsystem.setSwerveDrivePowers(xPower, yPower, angularPower);
         }, swerveSubsystem));
 
+        moverSubsystem.setDefaultCommand(new RunCommand(() -> {
+            double xPower = -mechController.getLeftX();
+            double yPower = -mechController.getRightY();
```

Comment:
this command doesn't seem to actually do anything besides save values to booleans

maybe for just seeing motor directions we can just make a simple subsystem to use temporarily